src/context.rs

```diff
@@ -381,6 +381,11 @@ impl Context {
             },
         };
 
+        let creation_pcrs = match creation_pcrs {
+            Some(val) => val,
```

Comment:
I had this same problem when I did Session where an Option<Session> that was none was supposed to be transformed into an ESYS_TR_NONE. I tried to make some attempts at using simple conversions but failed because I neither ESYS_TR nor Option<T> where my types. So I ended up declaring a static function belonging to the Session struct that makes these kind of conversions. 

I wonder if something similar should be made for PcrSelectionList?

Reply:
I think it depends on how many places there are that use this conversion - for `Session` it's used in lots of places, right? Don't know how common `PcrSelectionList` is

Reply:
I think it would be nice because it would make the approach consistent to have None is converted into TSS type.

Reply:
I'm ok with either approach, I'll let you decide

Reply:
I added `PcrSelectionList::list_from_option(pcr_list: Option<PcrSelectionList>) -> PcrSelectionList`.